doc/LanguageClient.txt

```diff
@@ -105,6 +105,13 @@ Default: >
         },
     }
 
+2.3 g:LanguageClient_diagnosticsMaxSigns  *g:LanguageClient_diagnosticsMaxSigns*
```

Comment:
Sorry for the pedantic, but please update the name to g:LanguageClient_diagnosticsSignsMax

Reply:
Thank you for the comments. I am happy to receiving any comments.
I updated the name..